integrated_channels/sap_success_factors/transmitters/learner_data.py

```diff
@@ -28,5 +32,13 @@ class SuccessFactorsLearnerDataTransmitter(SuccessFactorsTransmitterBase):
     (taken from OCNWebServicesConfiguration and OCNWebServicesEnterpriseCustomerConfiguration) and the oauth token.
 
     Record information about the success/failure of posting the learner data to SuccessFactors in
-     CompletionStatusEventAudit
+    the LearnerDataTransmissionAudit object provided, and save it to the database.
+
     """
+    def transmit(self, learner_data):
+        """
+        Transmit the learner data payload to the SAP SuccessFactors channel.
+        """
+        # TODO ENT-221 will handle transmitting the learner data payload to the channel
+        # Question: do we only want to transmit if learner_data.course_completed == True?
```

Comment:
For the SAP integration it is recommended that we only send a single completion event for each user, which means we'll need to wait until we are confident that the user has completed the course -- this can be difficult to do in certain scenarios, such as a self-paced course.